src/component/hoc/withLoading.tsx

```diff
@@ -4,7 +4,7 @@ import getDisplayName from "../../util/getDisplayName";
 import classNames from "classnames";
 
 export interface WithLoadingProps {
-    loading: boolean,
+    loading?: boolean,
```

Comment:
why loading is optional?

Reply:
The HOC API was causing typing issues. The effect is the same so it was simpler than trying to figure out how the API should look like...